templates/emails/collective.expense.missing.receipt.hbs

```diff
@@ -0,0 +1,30 @@
+Subject: {{fromCollective.name}} card charge expense is missing information
+
+
+{{> header}}
+
+<center>
+  <h1>
+    {{fromCollective.name}} card charge expense is missing information
+  </h1>
+</center>
+<p>
+  A {{currency expense.amount currency=expense.currency precision=2}} card charge was created and it is currently
+  missing critical information. For accounting reasons, your host will need an accurate description and a
+  copy of the receipt regarding this charge.
```

Comment:
A card attached to {{fromCollective.name}} was charged  {{currency expense.amount currency=expense.currency precision=2}} and we need you to input additional information. For accounting reasons, your Host will need an accurate description and a copy of the receipt.